### ISSUE TITLE
Try to request a package name transfer

### ISSUE THREAD

**Issue by Mogost** [rpkilby/jsonfield2#24]
According to [PEP 541](https://www.python.org/dev/peps/pep-0541/) It would be nice to request a transfer. Due to the fact that the project is abandoned by the original owner.

**Comment by rpkilby** [comment]
Hi @Mogost. I'll reach out to @dmkoch first to see if he'd be willing to transfer ownership.

**Comment by rpkilby** [comment]
Alright - sent him an email requesting he transfer ownership. Hopefully between that and @-ing him here, he'll see the request. 

If he doesn't respond in a week or so, I'll reach out to the PyPI team and start the PEP 541 process.

**Comment by dmkoch** [comment]
Hey Ryan — do you want me to transfer ownership to you?

Dan

**Comment by rpkilby** [comment]
Yes - that'd be appreciated. I'm also @rpkilby on PyPI. https://pypi.org/user/rpkilby/

**Comment by rpkilby** [comment]
I'm curious if there will be any issues transferring the project. Even though this is named `jsonfield2`, it's still a fork of `django-jsonfield`, so it may not go through. If that's the case, I'll have to delete/recreate this repo first.

**Comment by Mogost** [comment]
https://github.com/dmkoch/django-jsonfield/network
![image](https://user-images.githubusercontent.com/721963/70601918-461d6580-1c04-11ea-90d0-b9ebf2089b25.png)
No problems should arise. You have a fork from the last commit of the master of original repo. And you stick to the original version numbering.
In the general case, I think there are enough rights to pypi, on which you need to release all the missing versions.
What has already been released under the name jsonfield on pypi should not change.
for example, I use pip in --hash mode and if something changes, there can be problems (I think I'm not the only one).

**Comment by rpkilby** [comment]
> No problems should arise. You have a fork from the last commit of the master of original repo. And you stick to the original version numbering.

I should have been more clear. What I was wondering is if GitHub will have technical issues transferring the repo to my account. Typically, if a user has forked a repo, they need to delete the fork before the source repo can be transferred to them. This is at least in part due to a naming conflict - the fork and the source repo share the same name. In my case, the fork has a different name, but will GH allow me to have both a source repo and a fork under the same user? 

Curiosity got the better of me, and I just tested this out with a dummy org/repo.

> rpkilby already has a repository in the dummy/test network

It looks like I need to muck around and remove `jsonfield2` from the network.
